lib/drivers/docker/container.js

```diff
@@ -0,0 +1,277 @@
+'use strict';
+
+var _ = require('lodash');
+var debug = require('debug')('strong-pm:docker:container');
```

Comment:
I might be biased, because its the traditional unix default, but can we all agree to sort in ASCII order?